usaon_vta_survey/templates/macros/nav_buttons.j2

```diff
@@ -0,0 +1,17 @@
+{% macro nav_buttons(current_user) -%}
+  {% from "bootstrap5/nav.html" import render_nav_item %}
+
+  {{ render_nav_item("root", "Home")}}
```

Comment:
This button won't work because there is no "root" route, but I think creating that should be part of integrating this PR. Having the "Home" button point to the "surveys" route didn't make much sense to me :)

Maybe just an empty page that says "welcome!" for now?